src/dictionaries/en.json

```diff
@@ -1,3 +1,74 @@
-{
-  "hello": "hello"
-}
+
+{

```

Comment:
what is this?

Reply:
had resolve in another pr

Reply:
No, please dont change if you dont touch the file. Remove the changed please